pallets/phala/src/registry.rs

```diff
@@ -151,21 +158,27 @@ pub mod pallet {
 		PRuntimeNotFound,
 		// Additional
 		UnknownCluster,
+		Unsupported,
```

Comment:
I suggest `NotImplemented`

Reply:
I still prefer `NotImplemented` because its usage is replace `panic!("unimpleneted");`

Reply:
Okay, fixed